images/macos/templates/macOS-11.pkr.hcl

```diff
@@ -0,0 +1,242 @@
+variable "source_vm_name" {
+  type = string 
+  default = "clean_macos_11_300gb"
+}
+
+variable "source_vm_tag" {
+  type = string
+  default = "bigsur_300g"
+}
+
+variable "vm_name" {
+  type = string
+  default = "macos-11"
+}
+
+variable "vcpu_count" {
+  type = string
+  default = "4"
+}
+
+variable "ram_size" {
+  type = string
+  default = "6G"
+}
+
+variable "build_id" {
+  type = string
+}
+
+variable "macos_user" {
+  type = string
+}
+
+variable "macos_password" {
+  type = string
+}
+
+variable "xcode_user" {
+  type = string
+}
+
+variable "xcode_password" {
+  type = string
+}
```

Comment:
May be keep the same variable names like we have in current templates:
- vm_username
- vm_password
- xcode_install_user
- xcode_install_password